### PR TITLE
Thk v1.15.6

### PR DESCRIPTION
**Pull request by TomKimsey**

### CODE REVIEW

File: .github/workflows/package-test.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-# Compile Test Action for Maven-based Ewon ETK Projects
+# Package Test Action for Maven-based Ewon ETK Projects
 # Version: 2.1
 #
 # This action is configured to automatically run when a push
 # is made or pull request is merged to the `main` or `master`
 # branch.
 
-name: Compile Test
+name: Package Test
```
Comment (alexjhawk) [inline .github/workflows/package-test.yml:9]: What do you think about `Package Build Test` or `Packaging Test`? I don't think it's a huge concern, but `Package Test` just seems off to me.

Comment (TomKimsey) [reply .github/workflows/package-test.yml:9]: This is still in draft. not sure why i added a reviewer yet.

Im fine with "Package Test" or "Build Test", "Package Test" doesnt bother me at all.

File: pom.xml [hunk 2]
```diff
@@ -171,6 +171,7 @@
           <dependencySourceExcludes>
             <dependencySourceExclude>com.hms_networks.americas.sc.mvnlibs:ewon-etk:*</dependencySourceExclude>
           </dependencySourceExcludes>
+          <failOnWarnings>true</failOnWarnings>
```
Comment (alexjhawk) [inline pom.xml:174]: This should be carried upstream to the starter project.

File: .github/workflows/package-test.yml (hunk 1 above)
Comment (alexjhawk) [reply .github/workflows/package-test.yml:9]: I'm okay with either, although `Build Test` may be a bit more descriptive as it covers the Javadoc checking too.

**Review by alexjhawk — DISMISSED** [review]

File: pom.xml (hunk 2 above)
Comment (TomKimsey) [reply pom.xml:174]: Yeah I agree

**Review by alexjhawk — APPROVED** [review]

**Pull request merged**